Ldap/Converter.php

```diff
@@ -61,4 +61,20 @@ function ($matches) { return chr(hexdec($matches[1])); },
 
         return $string;
     }
+
+    /**
+     * Convert string to symfony role schema.
+     *
+     * @param string $role
+     *
+     * @return string
+     */
+    public static function strToSymRoleSchema($role)
+    {
+        $role = preg_replace('/\W+/', '_', $role);
+        $role = trim($role, '_');
+        $role = strtoupper($role);
```

Comment:
Could you apply this patch for UTF-8 compliant? https://github.com/BorisMorel/LdapBundle/pull/137